src/Resources/skeleton/Story.tpl.php

```diff
@@ -0,0 +1,12 @@
+import <?php echo $componentName; ?> from "../templates/<?php echo $template; ?>";
+import { twig } from '@sensiolabs/storybook-symfony-webpack5';
+
+export default {
+    component: (args) => ({
+        component: {<?php echo $componentName; ?>},
+        template: twig`
+            <twig:<?php echo $componentName; ?>/>
+        `,
+    })
+}
+
```

Comment:
It would be nice to add a default story here, like 

```js
export const Default = {};
```

This is required for Storybook to properly index the story file, otherwise we'll get an indexing error AFAIK.